src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/DocumentTransformerTest.kt

```diff
@@ -66,7 +68,7 @@ class DocumentTransformerTest {
   }
 
   @Test
-  fun `transformToApiFiltered filters out convictions other than one specified`() {
```

Comment:
no longer filtering by convictionId due to the 'Temporary Accommodation' not needed discovery (see above comment)